diff --git a/node_modules/expo-document-picker/ios/DocumentPickerModule.swift b/node_modules/expo-document-picker/ios/DocumentPickerModule.swift
index f591926..40f8884 100644
--- a/node_modules/expo-document-picker/ios/DocumentPickerModule.swift
+++ b/node_modules/expo-document-picker/ios/DocumentPickerModule.swift
@@ -130,7 +130,7 @@ public class DocumentPickerModule: Module, PickingResultHandler {
       try FileManager.default.copyItem(at: documentUrl, to: newUrl)
     }
 
-    let mimeType = self.getMimeType(from: documentUrl.pathExtension)
+    let mimeType = self.getMimeType(from: documentUrl.pathExtension) ?? "application/octet-stream"
 
     return DocumentInfo(
       uri: newUrl.absoluteString,
